src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -90,13 +90,28 @@ public void ConfigureServices(IServiceCollection services)
 
 #### List of metrics produced
 
-The instrumentation is implemented based on [metrics semantic
-conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#metric-httpserverduration).
-Currently, the instrumentation supports the following metric.
+The instrumentation is implemented based on
+[metrics semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/4bbb8c907402caa90bc077214e8a2c78807c1ab9/docs/http/http-metrics.md).
 
-| Name  | Instrument Type | Unit | Description |
-|-------|-----------------|------|-------------|
-| `http.server.duration` | Histogram | `ms` | Measures the duration of inbound HTTP requests. |
+Have you opt-ed into the new Http Semantic Conventions using `OTEL_SEMCONV_STABILITY_OPT_IN`?
+
+* If yes, the instrumentation supports the following metric.
+
+    | Name  | Instrument Type | Unit | Description |
+    |-------|-----------------|------|-------------|
+    | `http.server.request.duration` | Histogram | `s` | Measures the duration of inbound HTTP requests. |
+
+    This metric is emitted in "seconds" as per the semantic convention. While
+    the convention [recommends using custom histogram buckets](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration)
+    , this feature is not yet available in OTel .NET. However, the OTel SDK
```

Comment:
```suggestion
    , this feature is not yet available via .NET metrics API.  A workaround has been included in OTel SDK starting version `1.6.0` which applies recommended buckets by default for this metric.
```

Reply:
Also, this needs to be included in changelog: No action is needed from users targeting latest version of instrumentation library as that will bring in 1.6.0 SDK version by default.